src/main/java/core/basesyntax/ChangeToCar.java

```diff
@@ -5,5 +5,10 @@
  * и возвращает тип авто в зависимости от веса (если weight меньше 1500, то CAR, если больше, то
  * TRUCK). Метод должен принимать Integer и возвращать AutoType</p>
  */
-public class ChangeToCar {
+public class ChangeToCar implements ChangeToCarInt {
```

Comment:
You can implement common functional interfaces https://metanit.com/java/tutorial/9.3.php